### ISSUE TITLE
[iceprod2] better 404 page (Trac #1482)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1474]
Instead of a request error, generate a styled 404 page which preserves the base layout (so we can still see the menu).

Migrated from https://code.icecube.wisc.edu/ticket/1482
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:28", 
    "description": "Instead of a request error, generate a styled 404 page which preserves the base layout (so we can still see the menu).", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067088921308", 
    "component": "iceprod", 
    "summary": "[iceprod2] better 404 page", 
    "priority": "major", 
    "keywords": "", 
    "time": "2015-12-10T22:25:44", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20151218T16:52:30: 
* david.schultz changed status from "new" to "closed"
* david.schultz changed resolution from "" to "fixed"
* david.schultz commented:

> In changeset 140603:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="140603"
> better self-signed cert. proper messaging start order (so we register the service before the first request). nginx mime-type configuration. ssl by default. actual 404 page. other minor fixes. closes #1474
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:28: 
* olivas changed milestone from "IceProd 2.0" to ""
* olivas commented:

> Milestone IceProd 2.0 deleted
